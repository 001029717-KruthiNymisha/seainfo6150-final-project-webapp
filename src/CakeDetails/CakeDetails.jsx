import React from 'react'
import styles from "./CakeDetails.module.css";
import { Switch, Route, Link } from "react-router-dom";


const CakeDetails = props => {
  return (
  <div>
    <div>
    <h3 className={styles.titleDetails}>Cake Details</h3>
      </div>
       <div className = {styles.cakeDetails}> Summary: {props.Summary}<br /></div>
        <div className = {styles.cakeDetails}>Ingredients: {props.Ingredients}<br /></div>

    </div>

  );
};

export default CakeDetails;
