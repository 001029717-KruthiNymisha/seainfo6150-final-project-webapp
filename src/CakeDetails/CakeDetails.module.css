.cakeDetails {
  width: 500px;
  height: auto;
  margin-right:100px;
  margin-bottom: 10px;
  background-color: #FCDDD9;
  text-align: center;
  padding: 20px;
  font-size: 15px;
  color: black;
  font-family: cursive;
  box-shadow: 5px 5px 10px gray;
  border: 2px solid black;
}


.titleDetails{
text-decoration: underline;
font-family: cursive;
margin-left: 475px;
margin-right:100px;
margin-top:-40px;
color: Black;
}
@media(min-width: 400px){
.cakeDetails{
width: 300px;
height:auto;
margin-left:200px;
font-size:10px;
}
.titleDetails{
margin-left: 200px;
font-size:13px;
}

}
@media(min-width: 800px){
.cakeDetails{
width: 400px;
margin-left:300px;
font-size:13px;
}
.titleDetails{
margin-left: 300px;
font-size:14px;
}

}
@media(min-width: 1000px){
.cakeDetails{
width: 500px;
margin-left: 475px;
font-size:14px;
}
.titleDetails{
margin-left: 475px;
font-size:15px;
}

}
