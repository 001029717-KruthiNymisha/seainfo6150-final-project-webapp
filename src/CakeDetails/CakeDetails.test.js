import React from "react";
import { render } from "@testing-library/react";
import CakeDetails from "./CakeDetails.jsx";

it("renders the CakeDetails component correctly", () => {
  const { container } = render(
    <CakeDetails
      Summary="Sour cream coffee cake is hand made with real sour cream and butter, crunchy walnuts and a sprinkle of cinnamon for rich, fragrant flavor. Serve it for brunch or dessert with a hot cup of coffee or tea for a delightful treat."
      Ingredients="Granulated sugar, white rice flour, butter, sour cream, walnuts, eggs, powdered sugar, potato starch, brown sugar, tapioca flour, milk, vanilla, nonfat milk powder, baking powder, cinnamon, guar gum, salt, baking soda."
    />
  );
  expect(container).toMatchSnapshot();
});