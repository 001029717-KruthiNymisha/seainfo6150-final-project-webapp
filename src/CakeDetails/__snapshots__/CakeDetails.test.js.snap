// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CakeDetails component correctly 1`] = `
<div>
  <div>
    <div>
      <h3
        class="titleDetails"
      >
        Cake Details
      </h3>
    </div>
    <div
      class="cakeDetails"
    >
       Summary: 
      Sour cream coffee cake is hand made with real sour cream and butter, crunchy walnuts and a sprinkle of cinnamon for rich, fragrant flavor. Serve it for brunch or dessert with a hot cup of coffee or tea for a delightful treat.
      <br />
    </div>
    <div
      class="cakeDetails"
    >
      Ingredients: 
      Granulated sugar, white rice flour, butter, sour cream, walnuts, eggs, powdered sugar, potato starch, brown sugar, tapioca flour, milk, vanilla, nonfat milk powder, baking powder, cinnamon, guar gum, salt, baking soda.
      <br />
    </div>
  </div>
</div>
`;
